### PR TITLE
Basic checkout flow

### PR DESCRIPTION
**Pull request by bmigirl**
- Types are a big WIP, we will have some better typings for addresses coming from the api next week and I'd leave it be until then
- Styling is nonexistent here, as agreed, this is just for the flow

### CODE REVIEW

File: apps/checkout/src/index.tsx [hunk 1]
```diff
@@ -8,22 +8,22 @@ import { getCurrentRegion } from "./lib/regions";
 import { I18nProvider } from "@react-aria/i18n";
 import { createFetch, createSaleorClient, SaleorProvider } from "@saleor/sdk";
 
+// temporarily need to use @apollo/client because saleor sdk
+// is based on apollo. to be changed
+const saleorClient = createSaleorClient({
+  apiUrl: "https://new-checkout-flow-poc.api.saleor.rocks/graphql/",
```
[the remark this answers is not in the document]
Comment (bmigirl) [reply apps/checkout/src/index.tsx:14]: Added [in this pr](https://github.com/saleor/saleor-checkout/pull/26/files#diff-3b50857d24aed5f1defb5e0a3a757941da9fce99649a8123a7cb24c68c9bbc82R3)

File: apps/checkout/src/index.tsx [hunk 2]
```diff
@@ -8,22 +8,22 @@ import { getCurrentRegion } from "./lib/regions";
...
+  channel: "default-channel",
+});
+
 const authorizedFetch = createFetch();
 
 const client = createClient({
-  url: "https://latest.staging.saleor.cloud/graphql/",
+  url: "https://new-checkout-flow-poc.api.saleor.rocks/graphql/",
```
[the remark this answers is not in the document]
Comment (bmigirl) [reply apps/checkout/src/index.tsx:21]: Same as above

**Comment by Qasem-h** [comment]
@bmigirl @jwm0
saleor-checkout will merge to react-storefront ?

**Comment by bmigirl** [comment]
@Qasem-h actually not, checkout is a standalone app to be used independently from the storefront

**Comment by bmigirl** [comment]
Linters pass locally, ci commands are fixed [here](https://github.com/saleor/saleor-checkout/pull/26/files#diff-bc6c550346b5116243dd07db1c19f68e9356564b4489d86d1cb56d96e25e04bcR35)

**Review by jwm0 — APPROVED** [review]

**Pull request merged**